### ISSUE TITLE
Help me 

### ISSUE THREAD

**Issue by RutviShah97** [top-gg/issues#639]
I can't vote to any bot of discord.

**Comment by shivaco** [comment]
You were told to file a proper bug report and follow the issue template, yet you literally create the same issue with the same description that does not explain absolutely ***anything***. You can receive help in the [Discord server](https://discord.gg/dbl).

Side question: Is it that hard to follow a template?

**Comment by marcorennmaus** [comment]
Duplicate of #633, #634 and #638

Remember that the GitHub repo is meant to be track bugs and feature requests, for support please join the [Top.gg support server](https://discord.gg/dbl). Do not spam issues just because they keep getting closed.
